portfolio/src/main/java/com/google/sps/data/CalendarEvent.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+
+/** Models a calendar event. It can be used for both creation and import flow. */
+public class CalendarEvent {
+  private String name;
```

Comment:
Please indicate which fields are optional and which are required. And indicate which fields can contain null.

Reply:
Updated in constructor comment.